subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -33,134 +33,6 @@ Some plugins will break with this new version of Gradle, for example because the
 . Run `gradle wrapper --gradle-version {gradleVersion}` to update the project to {gradleVersion}.
 . Try to run the project and debug any errors using the <<troubleshooting.adoc#troubleshooting, Troubleshooting Guide>>.
 
-[[changes_7.6]]
-== Upgrading from 7.5 and earlier
-
-[[exported_key_deduplication]]
-=== Exported keys are now deduplicated
-
-link:dependency_verification.html#sec:local-keyring[Dependency verification's] way of key serialization was changed in 7.6: keys now are sorted and deduplicated.
-Forward compatibility was retained, but the following exports might cause a large VCS diff between the old, and the new keyfile.
-
-=== Updates to Attribute Disambiguation Rules related methods
```

Comment:
You are reverting way too many things in there. Only the section about export needs to be removed.

Reply:
Correct, hang on...

Reply:
PTAL